matrix_sdk/src/client.rs

```diff
@@ -727,11 +727,13 @@ impl Client {
     /// # Examples
     /// ```no_run
     /// use matrix_sdk::Client;
+    /// # use matrix_sdk_common::identifiers::ServerName;
+    /// # use std::convert::TryFrom;
     /// # use url::Url;
     /// # let homeserver = Url::parse("http://example.com").unwrap();
     /// # let limit = Some(10);
     /// # let since = Some("since token");
-    /// # let server = Some("server name");
+    /// # let server = Some(<&ServerName>::try_from("servername.com").unwrap());
```

Comment:
`"servername.com".try_into().unwrap()` should work I think (looks a bit less exotic :slightly_smiling_face:)